apps/public/src/app/(map)/locatie/kies/_components/Combobox/Combobox.module.css

```diff
@@ -0,0 +1,26 @@
+.combobox {
+  position: relative;
+  width: 100%;
+}
+
+.comboboxOptions {
+  background-color: white;
+  position: absolute;
+  width: 100%;
+
+  /*
+      Leaflet uses z-indexes to stack layers. The highest Leaflet z-index is 700. And the SidePanel uses 702.
+      https://leafletjs.com/reference.html#map-pane
+    */
+  z-index: 703;
+}
+
+.comboboxOptions ul {
+  margin-block-end: 0;
```

Comment:
I think we should have a margin at the end of the list, so users can see it's the end of the list, and not just cut of by the window.

Reply:
duplicate